src/features/loupe/DeleteImagesAlert.jsx

```diff
@@ -1,19 +1,43 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { deleteImages, setDeleteImagesAlertOpen, selectDeleteImagesAlertOpen } from '../images/imagesSlice.js';
+import {
+  deleteImages,
+  setDeleteImagesAlertOpen,
+  selectDeleteImagesAlertOpen,
+} from '../images/imagesSlice.js';
 import { selectSelectedImages } from '../review/reviewSlice.js';
-import { Alert, AlertPortal, AlertOverlay, AlertContent, AlertTitle } from '../../components/AlertDialog.jsx';
+import {
+  Alert,
+  AlertPortal,
+  AlertOverlay,
+  AlertContent,
+  AlertTitle,
+} from '../../components/AlertDialog.jsx';
 import Button from '../../components/Button.jsx';
 import { red } from '@radix-ui/colors';
+import { deleteImagesTask, fetchTask, selectDeleteImagesLoading } from '../tasks/tasksSlice.js';
+import { IMAGE_QUERY_LIMITS } from '../../config.js';
 
 const DeleteImagesAlert = () => {
   const dispatch = useDispatch();
   const open = useSelector(selectDeleteImagesAlertOpen);
   const selectedImages = useSelector(selectSelectedImages);
   const selectedImageIds = selectedImages.map((img) => img._id);
 
+  const deleteImagesLoading = useSelector(selectDeleteImagesLoading);
+
+  useEffect(() => {
+    if (deleteImagesLoading.isLoading && deleteImagesLoading.taskId) {
+      dispatch(fetchTask(deleteImagesLoading.taskId));
+    }
+  }, [deleteImagesLoading, dispatch]);
+
   const handleConfirmDelete = () => {
-    dispatch(deleteImages(selectedImageIds));
+    if (selectedImages.length <= IMAGE_QUERY_LIMITS[2]) {
```

Comment:
I think we should use declare separate limit in the `config.js` file for this. Something like `DELETE_IMAGE_LIMIT`. The `IMAGE_QUERY_LIMITS` is/was for trying different image limits in each page when we query for more images.